nl_open_data/user_config.toml

```diff
@@ -25,4 +25,5 @@
 # All data paths are relative to "Path.home()/root"
 root = "Projects/nl-open-data"
```

Comment:
Isn't it better to define everything relative to the main project folder? Then we don't need to include root in user_config.toml but use Path-object like in config.py

Reply:
Good idea - I'll take care of it.